static/src/javascripts/projects/common/modules/commercial/sticky-ad-banner.js

```diff
@@ -124,11 +124,12 @@ define([
         var transitionTimingFunction = 'cubic-bezier(0, 0, 0, .985)';
         var transitionDuration = '1s';
 
-        els.$header.css({
+        els.$page.css({
+            'will-change': 'transform',
```

Comment:
IIRC `will-change` [shouldn't be added at the time of animation](https://dev.opera.com/articles/css-will-change-property/#give-the-browser-enough-time-to-work) but should be added in anticipation of the event (and removed if that event never happens). Maybe in the render function you can add a threshold for adding the will-change and track scroll direction in the state? It should also be removed after the animation is complete. It's worth thinking about whether `will-change` is useful on `els.$adBanner` too as it's not just a replacement for the promoting to a layer hack but more a hint to the browser on *all* the things it can optimise for and get ready to change.